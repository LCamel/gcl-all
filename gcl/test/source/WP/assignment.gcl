{ True }
x := 1
{ 0 = x }