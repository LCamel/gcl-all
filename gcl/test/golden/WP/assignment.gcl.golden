### Result

[ Type Error ./test/source/WP/assignment.gcl:2:1-2
The definition x is not in scope ]

### Source

{ True }
x := 1
{ 0 = x }

### Trace

TraceLog "      [ save ] Parsed"
TraceLog "      [ save ] Converted"
